suricata.yaml.in

```diff
@@ -1180,6 +1180,10 @@ legacy:
 #   - reject
 #   - alert
 
+# Define maximum number of possible alerts that can be triggered for the same
+# packet. Default is 15
+#packet-alert-max: 15
```

Comment:
This needs a doc update.

Reply:
It's in the queue, just waiting for this first PR to be accepted :)

Reply:
Usually, I include the doc change with the PR containing the code changes